lib/moonchild.js

```diff
@@ -1,16 +1,24 @@
 'use strict';
 
-var _ = require('underscore'),
-    parser = require('./metadata'),
-    estraverse = require('estraverse'),
-    expanders = require('../third_party/expanders');
+var _                = require('underscore');
```

Comment:
I really don't like the style of lining up the `=`. Would you mind changing it? I'm fine with declaring each var on a separate line though -- I'm starting to prefer that style.